### PR TITLE
Resolve issue #454

### PR DESCRIPTION
**Pull request by cawerkenthin**
Resolve issue #454 - 8.1 endpoint

### CODE REVIEW

File: cmi5_spec.md [hunk 1]
```diff
@@ -476,10 +476,10 @@ The values for the URL launch parameters are described below:
   <tr><th colspan=2 align="left">endpoint</th></tr>
   <tr><th align="right" nowrap>Description:</th><td>A URL to the LMS listener location for xAPI requests to be sent to.</td></tr>
   <tr><th align="right" nowrap>LMS Usage:</th><td>The LMS MUST place the <strong><em>endpoint</em></strong> in the query string.</td></tr>
-  <tr><th align="right" nowrap>AU Usage:</th><td>The AU MUST get the <strong><em>endpoint</em></strong> value from the query string. The AU MUST use the <strong><em>endpoint </em></strong>value as the URL location to send xAPI requests to.</td></tr>
+  <tr><th align="right" nowrap>AU Usage:</th><td>The AU MUST get the <strong><em>endpoint</em></strong> value from the query string. The AU MUST use the <strong><em>endpoint </em></strong>value as the Base Endpoint for xAPI requests.</td></tr>
```
Comment (brianjmiller) [inline cmi5_spec.md]: Is there a reason to capitalize "Base Endpoint" here?

Comment (cawerkenthin) [inline cmi5_spec.md]: It is a defined term in xAPI spec.

**Comment by brianjmiller** [comment]
I'm 👍  on this PR.

**Comment by bscSCORM** [comment]
👍

**Pull request merged**